.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3.6
+      - image: viclim/elasticsearch:i14y
```

Comment:
Can you change this to simply `docker build -t elasticsearch:i14y` within the CI run? That way no hosting is required.